tests/test_dojo.py

```diff
@@ -74,7 +79,102 @@ def test_raise_error_on_duplicate_persons(self):
 		self.dojo.create_room("Hunter", "Office")
 		self.dojo.add_person("staff", "John", "Ngravo")
 		self.assertEqual(self.dojo.add_person("staff", "John", "Ngravo"), "Person already exists")
-			 
+
+class TesstPrintFunctions(unittest.TestCase):
+	"""A class to test the print functionalities in v1"""
+
+	def setUp(self):
+		self.dojo = Dojo()
+
+	def test_print_names_of_people_in_room(self):
+		#create rooms and add persons
+		self.dojo.create_room("Nairobi", "Office")
+		self.dojo.add_person("fellow", "ernest", "achesa", "N")
+		self.dojo.add_person("staff", "george", "wanjala")
+		#snapshot to store sys.stdout
+		temp = sys.stdout
+		#capture will hold the output being printed
+		capture = StringIO()
+		sys.stdout = capture
+		#call functions
+		self.dojo.print_room("Nairobi")
+		#end functions
+		#revert to snapshot of sys.stdout
+		sys.stdout = temp
+		#the printed value that was captured
+		output = capture.getvalue()
+		heading = "\x1b[33m\nROOM NAME: NAIROBI\tROLE\n"
+		expected_output = heading + "\x1b[0m\nErnest Achesa \t\tFELLOW\nGeorge Wanjala \t\tSTAFF\n"
+		self.assertEqual(output, expected_output)
+
+	def test_print_allocations_on_screen(self):
+		#create rooms and add persons
+		self.dojo.create_room("Nairobi", "Office")
+		self.dojo.create_room("Chetambe", "livingspace")
+		self.dojo.add_person("staff", "george", "wanjala", "Y")
+		#snapshot to store sys.stdout
+		temp = sys.stdout
+		capture = StringIO()
+		sys.stdout = capture
+		#call functions
+		self.dojo.print_allocations()
+		#end functions
+		#revert to snapshot of sys.stdout
+		sys.stdout = temp
+		output = capture.getvalue()
+		heading_1 = "\x1b[34m\nALLOCATIONS - OFFICES\x1b[0m\n"
+		heading_2 = "\x1b[33m\nROOM NAME: NAIROBI\tROLE\t\tACCOMODATION\n"
+		Office_occupants = "\x1b[0m\nGeorge Wanjala \t\tSTAFF\t\t\n"
+		heading_3 = "\x1b[34m\nALLOCATIONS - LIVING SPACES\x1b[0m\n"
+		heading_4 = "\x1b[33m\nROOM NAME: CHETAMBE\tROLE\t\tOFFICE\n\x1b[0m\n"
+		expected_output = heading_1 + heading_2 + Office_occupants + heading_3 + heading_4
+		self.assertEqual(output, expected_output)
```

Comment:
![image](https://cloud.githubusercontent.com/assets/25553421/23766937/b7e56e08-0517-11e7-932d-d47d08fbed23.png)
the alignment of the output could be a bit off. you might want to realign the output format